content/guides/guides/launching-browsers.md

```diff
@@ -164,9 +164,9 @@ tests.
 For example, your web application might _only_ be designed to work in a Chrome
 browser, and not inside the Electron browser.
 
-In the plugins file, you can filter the list of browsers passed inside the
-`config` object and return the list of browsers you want available for selection
-during `cypress open`.
+In the [setupNodeEvents](/guides/tooling/plugins-guide#Using-a-plugin) function,
```

Comment:
Whenever we're talking about the`config` object, we should actually link to the `/api/plugins/configuration-api` page:

```suggestion
In the [setupNodeEvents](/api/plugins/configuration-api) function,
```